test/test_modules.py

```diff
@@ -25,6 +25,13 @@ class TestModule(TestCase):
     precision = 1e-5
     rel_tol = 1e-5
 
+    def setUp(self):
+        TestCase.setUp(self)
```

Comment:
nit: super().setUp ?

Reply:
That's the first thing I tried :) But it fail with error:
```
TypeError: super(type, obj): obj must be an instance or subtype of type
```
I think it's due to the fact that we call instantiate_device_type_tests to create device special class'es

Reply:
Fair enough